### PR TITLE
Update on 3.4.0 branch 

### PR DESCRIPTION
**Pull request by chatelgu**
Hi @haiyangwu ,

@camilleoudot and I have committed some changes on your branch dev-3.4.0.

We are really interested in your sdk. We would like to work on the latest version of mediasoup client on Android. 

In this merge request we just make the project compile on our environnement (AndroidStudio Dolphin).
We also starting to modify the java binding on the new mediasoup client lib.

We would like to continue working on this to implement bindings of data channels.
We are really interested by your feedbacks so far (we may have missed something) and your thoughs on a possible future collaboration on this repo.

BR
@chatelgu and @camilleoudot

### CODE REVIEW

File: mediasoup-client/src/androidTest/jni/generated_mediasoupclient_jni/jni/Parameters_jni.h [hunk 1]
```diff
@@ -0,0 +1,116 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
```
Comment (haiyangwu) [inline mediasoup-client/src/androidTest/jni/generated_mediasoupclient_jni/jni/Parameters_jni.h:1]: why add this file in androidTest? seems no need.

File: mediasoup-client/src/main/java/org/mediasoup/droid/Device.java [hunk 2]
```diff
@@ -152,10 +167,11 @@ public RecvTransport createRecvTransport(
       @NonNull String id,
       @NonNull String iceParameters,
       @NonNull String iceCandidates,
-      @NonNull String dtlsParameters)
+      @NonNull String dtlsParameters,
+      @NonNull String sctpParameters)
```
Comment (haiyangwu) [inline mediasoup-client/src/main/java/org/mediasoup/droid/Device.java:171]: Let's keep API the same as libmediasoup. Add a separate `createSendTransport` with the `sctpParameters` parameter

```c++
		SendTransport* CreateSendTransport(
		  SendTransport::Listener* listener,
		  const std::string& id,
		  const nlohmann::json& iceParameters,
		  const nlohmann::json& iceCandidates,
		  const nlohmann::json& dtlsParameters,
		  const nlohmann::json& sctpParameters,
		  const PeerConnection::Options* peerConnectionOptions = nullptr,
		  const nlohmann::json& appData                        = nlohmann::json::object()) const;
		SendTransport* CreateSendTransport(
		  SendTransport::Listener* listener,
		  const std::string& id,
		  const nlohmann::json& iceParameters,
		  const nlohmann::json& iceCandidates,
		  const nlohmann::json& dtlsParameters,
		  const PeerConnection::Options* peerConnectionOptions = nullptr,
		  const nlohmann::json& appData                        = nlohmann::json::object()) const;
```

File: mediasoup-client/src/androidTest/jni/generated_mediasoupclient_jni/jni/Parameters_jni.h (hunk 1 above)
Comment (chatelgu) [reply mediasoup-client/src/androidTest/jni/generated_mediasoupclient_jni/jni/Parameters_jni.h:1]: Looks like Parameters jni stuf is only used in androidTest : in [Parameters.java](https://github.com/haiyangwu/mediasoup-client-android/blob/dev/mediasoup-client/src/androidTest/java/org/mediasoup/droid/data/Parameters.java) the AndroidTest code

Is it planned to be migrated to the main code ?

**Comment by chatelgu** [comment]
Hello, I just come back from vacations 🎃 
I made changes you asked for except for the Parameters stuff (see https://github.com/haiyangwu/mediasoup-client-android/pull/58#discussion_r1011544440)

**Comment by haiyangwu** [comment]
LGTM and merged, Thx @chatelgu @camilleoudot

**Pull request merged**